lib/widget/carousel.dart

```diff
@@ -149,7 +148,9 @@ class MyController extends BoxController {
   // for multi view this dispatch when clicking on a card
   EnsembleAction? onItemTap;
   EnsembleAction? onItemChange;
-  int selectedItemIndex = -1;
```

Comment:
So whoever used selectedItemIndex previously will have their page ceased to work? Please make sure our App Developers are aware of this so they can make changes.

Reply:
Yes, they need to change it to `selectedIndex`